docs/samples/explanation/aif/germancredit/server/model.py

```diff
@@ -47,7 +49,17 @@ def predict(
         return {"predictions": predictions.tolist()}
 
 
+parser = argparse.ArgumentParser(parents=[kserve.model_server.parser])
+parser.add_argument(
+    "--model_name",
+    default="german-credit",
+    help="The name that the model is served under.",
+)
+args, _ = parser.parse_known_args()
```

Comment:
This should't needed anymore as it is defined in the kserve model server

Reply:
removed